src/Servers/IIS/IIS/test/Common.FunctionalTests/ClientCertificateTests.cs

```diff
@@ -43,6 +43,7 @@ public Task HttpsNoClientCert_NoClientCert(TestVariant variant)
         }
 
         [ConditionalTheory]
+        [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/22319")]
```

Comment:
Minor question: Does WhatsBroken also no longer show quarantined tests? @BrennanConroy I thought the main issue was around finding new failures, not tracking the quarantined tests?

Reply:
WhatsBroken doesn't show anything anymore

Reply:
Worse than I thought ☹️ 

Guess issues will have to fill the gap for now. @ryanbrandenburg please do something distinctive w/ the issues you file so the leads can find them easily. A new label plus linking to them in your ops issue should be more than enough. Or, just assign them all to @Pilchie 😃